tests/verify_kzg_proof/small/verify_kzg_proof_case_commitment_not_in_G1/data.yaml

```diff
@@ -1,4 +1,4 @@
 input: {commitment: '0x8123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef',
-  z: '0x623ce31cf9759a5c8daf3a357992f9f3dd7f9339d8998bc8e68373e54f00b75e', y: '0x0200000000000000000000000000000000000000000000000000000000000000',
+  z: '0x623ce31cf9759a5c8daf3a357992f9f3dd7f9339d8998bc8e68373e54f00b75e', y: '0x00000000fffffffffe5bfeff02a4bd5305d8a10908d83933487d9d2953a7ed73',
```

Comment:
Were these changes done by running the new test gen from https://github.com/ethereum/consensus-specs/pull/3280/files ? Just wanted to make sure.

Reply:
Yes, I got them from re-runing the test gen from that branch. I also don't know why I got different tests.